src/js/services/ng-rankings.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * This is a service that can calculate the rankings.
+ */
+define('services/ng-rankings',[
+    'services/ng-services',
+    'services/ng-stages',
+    'services/ng-teams',
+    'services/ng-score'
+],function(module) {
+    "use strict";
+
+    return module.service('$rankings',
+        ['$stages','$teams', '$score',
+        function($stages, $teams, $score) {
+
+            const EMPTY = '---';
```

Comment:
Why do we need this? If it's going to be displayed, it should be the responsibility of that display component (e.g. display nothing, dashes, etc.)

If it is some kind of sentinel, then better use `undefined`, or `null` (the latter can be serialized to JSON, the former will cause the key to disappear).